src/operator/keycloak.ts

```diff
@@ -700,3 +724,103 @@ async function manageGroups(connection: KeycloakConnection) {
     console.error('Error in manageGroups: ', error)
   }
 }
+
+async function removeUserGroups(api: any, existingUser: any, teamGroups: string[]) {
```

Comment:
I am missing unit tests here

Reply:
I added unit tests in the keycloak.test.ts file for these functions.

Reply:
Fantastic!